docs/release-notes.asciidoc

```diff
@@ -7,6 +7,51 @@
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
 
+[discrete]
+[[release-notes-7.12.0]]
+== 7.12.0
+
+[discrete]
+[[features-7.12.0]]
+==== Features
+* Implements a connector for ServiceNow SIR ({pull}88190[#88190]).
+*  Implements the case's fields for the ServiceNow SIR connector.({pull}88655[#88655]).
+
+[discrete]
+[[bug-fixes-7.12.0]]
+==== Bug fixes and enhancements
+* Enables the Microsoft Team's action type for the detection engine ({pull}94239[#94239]).
+* Fixes bug for pre-populated endpoint exceptions ({pull}94025[#94025]).
+* Pushes ServiceNow ITSM comments on cases and alerts as work notes and improves error messaging ({pull}93916[#93916]).
+* Alert migrations can be finalized and cleaned up in all spaces ({pull}93809[#93809]).
+* Updates error handling logic to produce a cleaner message when deeply nested fields in KQL queries are greater than the default or what is set for the config property ({pull}93536[#93536]).
+* Updates shellcode telemetry for schema adjustment ({pull}93143[#93143]).
+* Fixes bug in the allowlist layout for security telemetry  ({pull}92850[#92850]).
+* Updates exceptions modal to use existing lists plug-in ({pull}92348[#92348]).
+* Moves PE details out of Ext context ({pull}92146[#92146]).
+* Fixes loading indicators in the rules management table ({pull}91925[#91925]).
+* Adds missing fields for security telemetry ({pull}91920[#91920]).
+* Fixes issues when pushing a case, that has alerts attached, to an external service ({pull}91638[#91638]).
+* Updates error banner when refreshing the rule status ({pull}91051[#91051]).
+* Fixes bug in the exceptions builder UI that causes invalid values to overwrite other values ({pull}90634[#90634]).
+* Fixes issues with searching the Exceptions list table by name ({pull}88701[#88701]).
+* Threshold rule fixes ({pull}93553[#93553])({pull}92667[#92667]).
+* Adds sub cases to the case list and a case details page ({pull}91434[#91434]).
+* Upgrades to use the IndexPatternService to get fields ({pull}91153[#91153]).
+* Adds new fields to the allowlist for alert telemetry ({pull}90868[#90868]).
+* Adds support for multiple `terms` aggregations within a Threshold Rule, as well as an additional `cardinality` aggregation for matching a specific number of unique values across a field. ({pull}90826[#90826]).
+* Introduces the network details and host details to the side panel. ({pull}90064[#90064]).
+* Adds ransomware exceptions  ({pull}89974[#89974]).
+* Extends the daily usage collection to include perf and run information on active security ML jobs. ({pull}89705[#89705]).
+* Reduces the detection engine's reliance on `_source` ({pull}89371[#89371]).
+* Pushes a new case to the connector when created ({pull}89131[#89131]).
+* Disallows JIRA labels with spaces ({pull}90548[#90548]).
+
+[discrete]
+[[features-7.12.0]]
+==== Known Issues
+* Pagination does not work in the All Cases table. ({pull}94929[#94929]).
+
```

Comment:
@nastasha-solomon change your ID tag to `known-issues-7.12.0` - that's why the build is failing. Also add the workaround for the pagination issue listed here: https://github.com/elastic/kibana/issues/94929#issuecomment-804212882